File: nekoshlink/shlink-auth-server/src/main/java/org/nekosoft/shlink/oauth2server/ShlinkOAuth2Server.java
package org.nekosoft.shlink.oauth2server;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class ShlinkOAuth2Server {

    public static final String VERSION_STRING = "1";

    public static void main(String[] args) {
        SpringApplication.run(ShlinkOAuth2Server.class, args);
    }

}

File: nekoshlink/shlink-auth-server/src/main/kotlin/org/nekosoft/shlink/oauth2server/ShlinkOAuth2Server.kt
package org.nekosoft.shlink.oauth2server

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

// https://docs.spring.io/spring-security/reference/servlet/authorization/method-security.html

@SpringBootApplication
class ShlinkOAuth2Server {

    companion object {
        const val VERSION_STRING = "1"
    }

}

fun main(args: Array<String>) {
    runApplication<ShlinkOAuth2Server>(*args)
}
